.github/workflows/deploy_website.yaml

```diff
@@ -0,0 +1,74 @@
+name: Deploy Website
+
+on:
+  pull_request:
+    types:
+      - closed # Trigger when the PR is closed
+    branches:
+      - 'main' # Target branch is main
+  workflow_dispatch:
+    inputs:
+      environment:
+        description: 'The environment used as target'
+        type: choice
+        required: true
+        default: prod
+        options:
+          - prod
+
+  # Allows external webhook trigger
+  repository_dispatch:
+    types:
+      - webhook
+
+# These permissions are needed to interact with GitHub's OIDC Token endpoint.
+permissions:
+  id-token: write
+  contents: read
+
+jobs:
+
+  cd_deploy:
+    name: Deploy to S3
+    # Trigger when the PR is merged
+    if: github.event_name == 'pull_request' && github.event.pull_request.merged == true
+    runs-on: 'ubuntu-latest'
+    strategy:
+      matrix:
+        environment: [ 'prod' ]
+
+    environment: ${{ matrix.environment }}
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@8e5e7e5ab8b370d6c329ec480221332ada57f0ab
+
+      - name: Deploy website
+        uses: ./.github/actions/deploy
+        with:
+          environment: ${{ matrix.environment }}
+          iam_role: ${{ secrets.DEPLOY_IAM_ROLE }}
+          bucket: ${{ vars.S3_BUCKET_NAME }}
+          cloudfront_distribution_id: ${{ vars.CLOUDFRONT_DISTRIBUTION_ID }}
+          strapi_token: ${{ secrets.STRAPI_TOKEN }}
+          strapi_api_url: ${{ vars.STRAPI_URL }}
+
+  manual_deploy:
+    name: Deploy to S3 (manual trigger) - (${{ inputs.environment }})
+    if: github.event_name == 'workflow_dispatch'
+    runs-on: 'ubuntu-latest'
+    environment: ${{ inputs.environment }}
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@8e5e7e5ab8b370d6c329ec480221332ada57f0ab
+
+      - name: Deploy website
+        uses: ./.github/actions/deploy
+        with:
+          environment: ${{ inputs.environment }}
+          iam_role: ${{ secrets.DEPLOY_IAM_ROLE }}
```

Comment:
Can you create a PR to create the IAM ROLE that we should use here? It should have limited permission, you can check [this as example](https://github.com/pagopa/developer-portal/blob/main/.infrastructure/40_iam.tf#L3-L78)

Reply:
Yes, now I create branches and PRs for role creation

Reply:
ok, i have open a PR https://github.com/pagopa/b2b-portals/pull/58 for create new role:

1) GitHubActionDeployECS for deploy CMS on ECS. In Workflow i have write secrets.IAM_ROLE_DEPLOY_CMS
2) GitHubActionDeployWebsite for deploy a static website on S3. In Workflow i have write secrets.IAM_ROLE_DEPLOY_WEBSITE